index.html

```diff
@@ -30,5 +30,82 @@ <h2>I’am a Software Developer</h2>
+        <div class="title">My Recent Works</div>
+        <hr>
+    </div>
+    <div class="multipost">
+        <img src="images/imgplaceholder.jpg" alt="Multi-Post">
+        <div class="multipost_title">Multi-Post Stories</div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="tags">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
+    <div class="cards">
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button  type="button">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button  type="button">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button  type="button">See Project</button>
+        </div>
+        <div class="card">
+            <div class="card_container">
+                <div class="card_title">Profesional Art Printing Data</div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </div>
+    </div>    
+</div>
+<div class="aboutme">
+    <div class="aboutme_title">About me</div>
```

Comment:
Kindly. Titles should be H1, H2, H3, etc. tags. (Depending on the importance on your site), this will make your site more accessible.